### PR TITLE
Explicitely document methods as to whether they deal with chunk or block coordinates.

### PR DESCRIPTION
**Pull request by cybertiger**
Some methods in World and Chunk are confusing as to whether they deal in terms of chunks or blocks, this should be documented.

### CODE REVIEW

**Comment by SagaciousZed** [comment]
This is more descriptive and a step in the right direction, but the changes need to be grammatically correct. You should also incorporate the note into the description.

**Comment by cybertiger** [comment]
The notes are only in the descriptions (though they could do with a <p> tag so they are in a separate line).

The changes are grammatically correct, if a little jarring. In the context of Minecraft blocks and chunks are valid units. I'd welcome some less negative criticism with actual suggestions on how to improve it.

**Comment by cybertiger** [comment]
Parhaps you can answer this one for me.

Javadoc's short description is split out by finding the end of the first sentence, why in the javadoc are there a whole load of missing periods at the end of the first sentence. Also why do you use &lt;p /&gt; instead of &lt;p&gt;. Oracle's Javadocs notes state you you should &lt;p&gt; here: http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format

**Comment by SagaciousZed** [comment]
Missing periods is only one of the many reasons that the JavaDoc are in need of improvement. As for the convention that we currently use, it is merely the status quo ante. That having been said, under the html specification, the use of the self closing paragraph tag makes no sense. It makes sense to revisit the issue.

EDIT: we will now use the proper html syntax when denoting paragraphs. In html 4.01 the closing tag is optional and we are keeping it at such.

File: src/main/java/org/bukkit/World.java [hunk 1]
```diff
@@ -97,10 +97,10 @@
     public Block getHighestBlockAt(Location location);
 
     /**
-     * Gets the {@link Chunk} at the given coordinates
+     * Gets the {@link Chunk} at the given coordinates.
      *
-     * @param x X-coordinate of the chunk
-     * @param z Z-coordinate of the chunk
+     * @param x X-coordinate in chunks of the chunk
+     * @param z Z-coordinate in chunks of the chunk
```
Comment (Wolvereness) [inline src/main/java/org/bukkit/World.java:103]: chunks of the chunk??

**Comment by Wolvereness** [comment]
This is going to need to be rebased for all of the `<p>` tag changes.

File: src/main/java/org/bukkit/World.java (hunk 1 above)
Comment (cybertiger) [inline src/main/java/org/bukkit/World.java:103]: Yeh, I know, I hate it too, please, suggest a better alternative, which conveys the unit and doesn't make me want to blow chunks. I guess "?-Coordinate in chunks, of the chunk" is a slight improvement, I still hate it.

**Comment by cybertiger** [comment]
sure, no issues rebasing and squashing as/when necessary.

File: src/main/java/org/bukkit/World.java (hunk 1 above)
Comment (SagaciousZed) [inline src/main/java/org/bukkit/World.java:103]: ....in unit of chunks....

**Comment by SagaciousZed** [comment]
Do you plan on updating this @cybertiger ?

**Comment by cybertiger** [comment]
Not a productive use of my time, sorry, I'll close it.

**Pull request closed without merging**